### DISCUSSION TITLE
Print Profiles missing

### DISCUSSION THREAD

**Discussion by DavidG-Aus** [supermerill/SuperSlicer#3673] [Q&A]
I have been using SS for months and love it.  Today I opened SS and only the 'default' print profile was available. ??
The .ini files for all print profiles are in the config directory (Win10). 
I restarted SS a couple of times and **once** all the profiles were visible.
![image](https://user-images.githubusercontent.com/35384963/221729445-831a5be5-74ac-45d2-b317-5098bc07a472.png)

**Comment by x-c-dog-x** [comment]
What happens to the list if you click the green compatibility flag off on the toobar between the Help '?' and the Save Setting disk icon?

**Reply by DavidG-Aus** [reply]
I don't see a green flag where you indicate.  There is a green flag at the left end of the toolbar but it seems to act the same as the dropdown arrow at the right of the selected profile.

![image](https://user-images.githubusercontent.com/35384963/222933640-124c2aee-f7ac-4ae8-a2b5-9711f97ae39f.png)

**Comment by x-c-dog-x** [comment]
<img width="631" alt="Screenshot 2023-03-06 073024" src="https://user-images.githubusercontent.com/61723187/223124672-68f6db9f-74f5-40f9-bb83-b698f9097b33.png">

That may be part of the issue.  Clicking the Compatiblity Fag changes it from green to red and then allow you to see the what SuperSlicer believes are setting that wouldn't work for your particular printer.  This is not always true. By default if I set up PrusaSlicer or SuperSlicer to include every filament when setting up both a  Prusa FDM  printers and Voron printer on the same copy, neither program will show all of the filaments when a Voron printer is selected. Even though it shows all of the filaments when set to a Prusa  printer. Often I have set compatiblity to off and save the setting to a new profile. Then they become green flagged and show with default green flag lists. Works this wasy for Print Settings too.

**Reply by DavidG-Aus** [reply]
This Compatibility flag is not visible on my display (as shown above). The issue is not resolved and it is starting to be a real problem. Any other suggestions?

**Comment by DavidG-Aus** [comment]
Ok I think I am onto something.  I changed the Filament Settings to my original 0.4mm nozzle filament and then the green Compatibility flag appeared. I clicked it and all my printer profiles showed up with red flags (ie incompatible).  There is something about all this that I do not understand!!  Why do the printer profiles depend on the filament profiles?  And why are all these printer profiles showing as incompatible when I have selected my original filament profile?
